src/Backend/Executor.ts

```diff
@@ -22,7 +22,7 @@ interface Executor {
   toolchains(): Toolchains;
 
   // inference a job with a cmd
-  inference(): Command;
+  inference(_modelPath: string, _options?: string[]): Command;
```

Comment:
For now, modelpath and options have been added to `inference`, but it will be substituted by `cfg` file path as `compile` after `one-infer` landed.

Reply:
Great. You can add `TODO: Introduce cfg instead of other params` when you push this on main branch.

Reply:
@yunjayh 
when you change those params in future PR,
could you also change the name of method to `infer` or `runInference`?
ref: https://google.github.io/styleguide/jsguide.html#naming-method-names

Reply:
Thank you for your advice, all!